Documentation/specs/runtime-configuration-file.md

```diff
@@ -2,206 +2,155 @@
     * `System.GC.Server` (old: `gcServer`) - Boolean indicating if the server GC should be used (Default: `true`).
     * `System.GC.Concurrent` (old: `gcConcurrent`) - Boolean indicating if background garbage collection should be used.
-* `framework` - Indicates the `name`, `version`, and other properties of the shared framework to use when activating the application. The presence of this section indicates that the application is a portable app designed to use a shared redistributable framework.
-* `applyPatches` - When `false`, the framework version is strictly obeyed by the host. When `applyPatches` is unspecified or specified as `true`, the framework from either the same or a higher version that differs only in the `SemVer` patch field will be used.
+* `framework` - Indicates the `name`, `version`, and other properties of the shared framework to use when activating the application including `applyPathes` and `rollForwardOnNoCandidateFx`. The presence of this section indicates that the application is a framework-dependent app.
+* `applyPatches` - When `false`, the framework version is strictly obeyed by the host. When `applyPatches` is unspecified or `true`, the framework from either the same or a higher version that differs only by the patch field will be used.
   * For example, if `version=1.0.1` and `applyPatches` is `true`, the host would load the shared framework from `1.0.{n}`, where `n >= 1`, but will not load from `1.1.0`, even if present. When `applyPatches` is `false`, the shared framework will be loaded from `1.0.1` strictly.
-  * **Note:** This does not apply to `SemVer`'s `prerelease` versions, but only for `production` releases.
-  * **Note:** This section will not be used for standalone applications that do not rely upon a shared framework.
-
-* Others _TBD_
+  * **Note:** This does not apply to `pre-release` versions; it applies only to `production` releases.
+  * **Note:** This section should not exist self-contained applications because they do not rely upon a shared framework.
+* `rollForwardOnNoCandidateFx` - Determines roll-forward behavior of major and minor. Only applies to `production` releases. Values: 0(Off), 1 (roll forward on [minor]), Roll forward on [major] and [minor]
+ See [roll-forward-on-no-candidate documentation](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/roll-forward-on-no-candidate-fx.md) for more information.
 
-These settings are read by `corehost` to determine how to initialize the runtime. All versions of `corehost` **must ignore** settings in this section that they do not understand (thus allowing new settings to be added in later versions).
+These settings are read by `apphost` to determine how to initialize the runtime. All versions of `apphost` **must ignore** settings in this section that they do not understand (thus allowing new settings to be added in later versions).
```

Comment:
IMO we shouldn't use "apphost" to mean both the `apphost.exe` and `dotnet.exe`.  Can we just say "host" when we mean them both?

Reply:
Done